hxd/fs/Convert.hx

```diff
@@ -1,48 +1,114 @@
 package hxd.fs;
 
+/**
+	The base class that processes raw asset file into a compatible with the engine format.
+
+	@see [Resource baking](https://github.com/HeapsIO/heaps/wiki/Resource-Baking) wiki entry.

```

Comment:
If you promise to keep heaps.io at least somewhat up to date ;) Referring to wiki gives me a lot less headaches in a long run, since I would see less people confused by old version of the wiki on the website.

Reply:
I think it's supposed to be automatically sync. The problem we have atm is that the API documentation relies on released heaps version (which I will to today)

Reply:
Vast majority of our tiny community uses git version, and for newcomers "switch to git version" is among the first things that are recommended. Especially when we see error logs pointing at `haxelib/heaps/1.8.0 `.

Reply:
Yeah I should instead change documentation building to use heaps GIT instead of released version.